### DISCUSSION TITLE
Another C# library of stock indicators

### DISCUSSION THREAD

**Discussion by cryptocoinserver** [DaveSkender/Stock.Indicators#690] [Dev notes]
One of the coders I follow on TradingView recently shared his C# .NET based project:
[ https://github.com/ooples/OoplesFinance.StockIndicators](https://github.com/ooples/OoplesFinance.StockIndicators)

_"This is a stock indicator library that is completely open source and very easy to use. Current version contains 392 stock indicators with over 700 planned for future releases."_

Thought this might be interesting for you. Maybe to cooperate or something like that.

**Comment by DaveSkender** [comment]
I suspect he's aware of us and just wanted to do his own thing.  I'd welcome @ooples to join us if interested, but I suspect he might have different design goals and objectives.  I respect his creativity and dedication to the open-source community.

**Reply by ooples** [reply]
I did take a look at your code first and it didn't match what I was wanting to implement. I have my own personal library that I have spent the last few years putting together and all I'm doing is changing the format a bit and then making them public. It was just much easier for me to slightly change the format of what I already had rather than changing everything to the format you have. That being said I would love to collaborate and would be willing to be a contributor to this project if you will have me

Thanks,
Franklin

**Reply by DaveSkender** [reply]
Franklin, sounds like we have a lot of common interests, and what Markus describes, it sounds like you have a clear passion for this subject.  Let’s do a virtual coffee to discuss further.  Direct message me at [@daveskender](https://twitter.com/messages/compose?recipient_id=27475431) and let’s setup a time!

**Reply by ooples** [reply]
I'm not on social media but can be reached on telegram @cheatcountry or email cheatcountry@gmail.com
